### PR TITLE
Update Rachael's Speaker bio

### PR DESCRIPTION
**Pull request by ChaelCodes**
Update speaker bio for Rachael on the home page, and prefer GitHub to LinkedIn.

* **Name:** Rachael Wright-Munn
* **Title:** Sr.Developer Relations Engineer
* **Twitter (Optional):**
* **Email (Optional):**
* **Company (Optional):**

### CODE REVIEW

File: website/_layouts/home.html [hunk 1]
```diff
@@ -31,8 +31,8 @@ <h5><a href="https://linkedin.com/in/thejonanshow">LinkedIn</a> | <a href="https
    <div class="column card">
       <img src="/assets/img/Rachael_headshot.jpg"></img>
       <br>
-      <h5>Rachael is a Senior Developer Relations Engineer at New Relic. She has been building software and web apps since 2012.</a></h5>
-      <h5><a href="https://www.linkedin.com/in/rachael-wright-chaelcodes/">LinkedIn</a> | <a href="https://twitter.com/ChaelCodes">Twitter</a> | <a href="https://twitch.tv/ChaelCodes">Twitch</a> | <a href="https://chael.codes">Personal</a> </h5>
+      <h5>Rachael wants to share the joy and beauty of programming through programming games and regular open-source streams on Twitch. She's been a professional dev since 2012, 3x Team Lead, and manages an open-source repo for playing games of Monster of the Week. Currently, she works as a Developer Relations Engineer for New Relic.</a></h5>
+      <h5><a href="https://github.com/thejonanshow">GitHub</a> | <a href="https://twitter.com/ChaelCodes">Twitter</a> | <a href="https://twitch.tv/ChaelCodes">Twitch</a> | <a href="https://chael.codes">Personal</a> </h5>
```
Comment (muydanny) [inline website/_layouts/home.html]: Hey @ChaelCodes did you want to link to Jonans GitHub?

Comment (ChaelCodes) [inline website/_layouts/home.html]:
```suggestion
      <h5><a href="https://github.com/ChaelCodes">GitHub</a> | <a href="https://twitter.com/ChaelCodes">Twitter</a> | <a href="https://twitch.tv/ChaelCodes">Twitch</a> | <a href="https://chael.codes">Personal</a> </h5>
```

Comment (ChaelCodes) [reply website/_layouts/home.html]: LOL. Sorry about that. Should be fixed.

**Review by muydanny — APPROVED** [review]
Great, thanks Rachael!

**Pull request merged**